proto/Cargo.toml

```diff
@@ -22,6 +22,7 @@ serde = { version = "1.0.203", features = ["derive"] }
 [build-dependencies]
 anyhow = "1.0.86"
 prost-build = "0.12.6"
+protox = "0.6.1"
```

Comment:
I've noticed that latest protox version is 0.7.1. Could you update, or is there a particular reason to not use it?

Reply:
That would require upgrading Prost, which breaks the build. I assumed you guys intentionally locked prost to this version